src/main.rs

```diff
@@ -235,7 +235,11 @@ fn main() {
                                 .help("Marks the end of a section excluded from branch coverage. The current line is part of this section.")
                                 .long("excl-br-stop")
                                 .value_name("regex")
-                                .takes_value(true))
+                                 .takes_value(true))
+
+                            .arg(Arg::with_name("demangle")
+                               .help("Demangle cpp and Rust symbols name")
+                               .long("demangle"))
```

Comment:
What if we made this true by default?

Reply:
I'm fine with a true by default, so let's rename this option into `no-demangle` or something else.